nn_with_simsiam.py

```diff
@@ -209,8 +208,40 @@ def configure_optimizers(self):
         scheduler = torch.optim.lr_scheduler.CosineAnnealingLR(optim, max_epochs)
         return [optim], [scheduler]
 
+class NNCLRModel(BenchmarkModule):
```

Comment:
Is there a reason why this class is not in the file `nnclr_model.py`? Furthermore, I do not get the difference between the two classes `NNCLRModel` and 'NNCLR'.

If the NNCLRModel truly only is the CLRModel, but with a different training step, could it inherit from it and only overwrite the training step? This would save a lot of code.